packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -329,6 +330,16 @@ class WebView extends StatefulWidget {
   /// The default policy is [AutoMediaPlaybackPolicy.require_user_action_for_all_media_types].
   final AutoMediaPlaybackPolicy initialMediaPlaybackPolicy;
 
+  /// Whether to use `AndroidViewSurface` and `SurfaceAndroidViewController` to create the widget.
+  ///
+  /// This flag is temporary and is highly subject to removal. One should only
+  /// use this if they are willing to test [WebView] features that aren't
+  /// available in the standard version and acknowledges that it is possible
+  /// this can have slower performance and/or unpredictable bugs.
```

Comment:
I don't think it has slower performance necessarily. It depends on what is being measured and how relevant that is. 

For example, this mode makes the webview itself much faster. The Flutter UI might be slower, but only on devices below Android 10. Only while the webview is rendered, and Flutter is producing a ton of frames. e.g. animations.   That's really it.  For the most part, these are pretty good trade-off for webview. 


I'd change this description to point out that it's an experimental feature that appends the webview to the Android view hierarchy for better performance and compatibility.

That's it for now. Let me know what you think.

Reply:
Also, re: unpredictable bug.  This is true for any new features, and it's definitely true for the current webview, which is the motivation behind this feature anyways.   I think I will rephrase it as "this mode fixes many issues while interacting with the Android webview. For more, see https://github.com/flutter/flutter/issues/61133"